hubblestack/utils/osquery_lib.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import os
+import salt.modules.cmdmod
+import json
+
+__salt__ = {'cmd.run': salt.modules.cmdmod._run_quiet,
+            'cmd.run_all': salt.modules.cmdmod.run_all}
+
+log = logging.getLogger(__name__)
+
+def query(query_sql='', osquery_path=None, args=None, max_file_size=104857600, timeout=10000):
+  try:
+    if not query_sql:
+      return ''
```

Comment:
Should return boolean False or null in case of these error cases.

If this is some practice of writing grains, then its ok.